seqr/views/apis/anvil_workspace_api.py

```diff
@@ -141,15 +150,25 @@ def validate_anvil_vcf(request, namespace, name, workspace_meta):
     # Validate the data path
     bucket_name = workspace_meta['workspace']['bucketName']
     data_path = 'gs://{bucket}/{path}'.format(bucket=bucket_name.rstrip('/'), path=path.lstrip('/'))
+    error = None
+    file_to_check = data_path
     if not data_path.endswith(VCF_FILE_EXTENSIONS):
         error = 'Invalid VCF file format - file path must end with {}'.format(' or '.join(VCF_FILE_EXTENSIONS))
-        return create_json_response({'error': error}, status=400, reason=error)
-    if not does_file_exist(data_path, user=request.user):
+    elif '*' in data_path:
+        try:
+            sharded_file_list = get_gs_file_list(data_path, request.user, check_subfolders=False)
+            file_to_check = sharded_file_list[0]
+        except Exception as ee:
+            error = f'Loading VCF file list for {data_path} failed with error {str(ee)}'
+            logger.error(error, request.user)
```

Comment:
It creates an error log because it can happen only when the frontend is hacked, or there are bugs in the code.